tests/functional/n/no/no_member.py

```diff
@@ -44,5 +44,10 @@ class Derived(Base):
 print(Derived.label)
 
 
+# Regression test for https://github.com/pylint-dev/pylint/issues/6094
+# pylint: disable-next=unnecessary-dunder-call
+print(parse.__getattribute__("urlparse"))
```

Comment:
```suggestion
print(parse.__getattribute__("urlparse"))  # [no-member] known false positive (#6094)
```

We'd also need to upgrade the txt and remove the fragment but you get the idea.

Reply:
I don't think that adds much benefit compared to just keeping the issue open in the issue tracker. Somebody who wants to add a test for it will very likely come up with the same test in a new PR.